app/views/documents/edit.html.erb

```diff
@@ -33,14 +32,32 @@
 
 <script src="https://js.shubox.io/v1/<%= Rails.application.secrets.shubox_js_id %>.js"></script>
 
+<script id="template" type="text/x-handlebars-template">
+<%= render "document", url: nil %>
+</script>
+
 <script>
 new Shubox("#document-drop", {
```

Comment:
would it make sense to include this JS in its own js file that is included here?